.github/workflows/linux_ci.yml

```diff
@@ -31,8 +31,19 @@ jobs:
           python-version: ${{ matrix.python-version }}
           architecture: ${{ matrix.architecture }}
 
-      - name: Install dependencies and test ONNX models
+      - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
           python -m pip install onnx==${{ matrix.onnx-version }} requests
-          python .azure-pipelines/test_models.py
\ No newline at end of file
+          python -m pip install onnxruntime
+          git clone https://github.com/microsoft/onnxruntime.git
+      - name: Test new models by onnx
+        run: |
+          export PYTHONPATH="$(pwd)/onnxruntime/tools/python/:$PYTHONPATH"
+          python workflow_scripts/test_models.py onnx
+      - name: Test new models by onnxruntime
```

Comment:
I decoupled onnx and onnxruntime in CI. There might be other things we can decouple like testing the uploaded onnx model file and testing the model from uploaded .tar.gz